packages/@aws-cdk/aws-elasticloadbalancingv2-targets/lib/alb-target.ts

```diff
@@ -23,7 +24,7 @@ export class AlbArnTarget implements elbv2.INetworkLoadBalancerTarget {
     return this.attach(targetGroup);
   }
 
-  private attach(_targetGroup: elbv2.ITargetGroup): elbv2.LoadBalancerTargetProps {
+  protected attach(_targetGroup: elbv2.ITargetGroup): elbv2.LoadBalancerTargetProps {
```

Comment:
`~` (Nit) - An alternative would be to make this `public _attach` and mark it as `@internal`. That would prevent it showing up in the docs and (potentially) confusing people who think they should be using it directly.